### DISCUSSION TITLE
HTTPS calls are not registered by mitmproxy on MacOS

### DISCUSSION THREAD

**Discussion by toddysm** [mitmproxy/mitmproxy#6024] [Intercept Everything!]
#### Problem Description
I have mitmproxy set up on a MacOS. Everything works fine but calls from two tools do not appear in the mitmproxy logs. The two tools are [notation](https://github.com/notaryproject/notation) and [oras](https://github.com/oras-project/oras). I am trying to understand the call flows for those tools to Docker Hub registry. Docker CLI calls are logged. Istn's mitmproxy be supposed to be systemwide?

#### Steps to reproduce the behavior:
1. oras push command doesn't get logged

#### System Information
Paste the output of "mitmproxy --version" here.
Mitmproxy: 9.0.1
Python:    3.11.2
OpenSSL:   OpenSSL 1.1.1t  7 Feb 2023
Platform:  macOS-13.2.1-arm64-arm-64bit

**Comment by Prinzhorn** [comment]
> Istn's mitmproxy be supposed to be systemwide?

Depends on which mode you're running, see https://docs.mitmproxy.org/stable/concepts-modes/

Does oras even use HTTP? Does it support a proxy? Did you configure it to use mitmproxy? I'm going to move this to discussions as this doesn't sound like a bug in mitmproxy.

**Reply by toddysm** [reply]
I have it configured as a regular proxy. And yes, oras uses HTTPS. Whether ORAS supports proxy is a good question :) But if I have the proxy configured on the OS level, shouldn't all network calls go through it?

Looking at the modes, it seems I should use `transparent` in my case. Giving it a try.
